samples/keys/identity_key_generation/README.rst

```diff
@@ -0,0 +1,72 @@
+.. _identity_key_generate:
+
+Identity key generation
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to generate a random device-specific identity using :ref:`lib_identity_key`, which is then stored in the Key Management Unit (KMU).
+
+Requirements
+************
+
+The following development kits are supported:
+
+.. table-from-sample-yaml::
+
+The :ref:`lib_hw_unique_key` library is required to generate and store the prerequisite Master Key Encryption Key (MKEK) into KMU.
+
+.. note::
+   Erase only the code pages when programming the device after the required identity key has been provisioned on the device, as ERASEALL removes the identity key from the system.
```

Comment:
```suggestion
   Erase the code pages when programming the device only after the required identity key has been provisioned on the device, as ERASEALL removes the identity key from the system.
```
Right?

Reply:
This is not accurate I think. We want to say here that not everything should be erased, only the code pages need to be erased. As you phrased it it sounds to me more like the " Only after the required identity key has been provisioned on the device, erase the code pages".

What we want to emphasize here is that only the code pages need to be erased. 

How about this:

"After the required identity key has be provisioned on the device only code pages should be erased, as ERASEALL will remove the identity key from the system."

Reply:
I see! :)
Now the sentence makes sense, but the new suggestion is better.
How about tweaking it into this:
"Once the required identity key is provisioned on the device, only the code pages should be erased as ERASEALL removes the identity key from the system.